0_meal_identification/meal_identification/notebooks/1.01-jg-GMMHMM-implementation.ipynb

```diff
@@ -34,6 +34,7 @@
    ]
   },
   {
```

Comment:
If the content is identical, please rollback the change

Reply:
fixed